src/Compilers/CSharp/Portable/BoundTree/UnboundLambda.cs

```diff
@@ -106,15 +106,15 @@ internal BoundLambda WithInAnonymousFunctionConversion()
         public TypeWithAnnotations GetInferredReturnType(ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo, out bool inferredFromFunctionType)
         {
             // Nullability (and conversions) are ignored.
-            return GetInferredReturnType(conversions: null, nullableState: null, ref useSiteInfo, out inferredFromFunctionType);
+            return GetInferredReturnType(conversions: null, nullableState: null, getterNullResilienceData: null, ref useSiteInfo, out inferredFromFunctionType);
         }
 
         /// <summary>
         /// Infer return type. If `nullableState` is non-null, nullability is also inferred and `NullableWalker.Analyze`
         /// uses that state to set the inferred nullability of variables in the enclosing scope. `conversions` is
         /// only needed when nullability is inferred.
         /// </summary>
-        public TypeWithAnnotations GetInferredReturnType(ConversionsBase? conversions, NullableWalker.VariableState? nullableState, ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo, out bool inferredFromFunctionType)
+        public TypeWithAnnotations GetInferredReturnType(ConversionsBase? conversions, NullableWalker.VariableState? nullableState, NullableWalker.GetterNullResilienceData? getterNullResilienceData, ref CompoundUseSiteInfo<AssemblySymbol> useSiteInfo, out bool inferredFromFunctionType)
```

Comment:
consider updating the docs here.  To me, the name `GetterNullResilienceData getterNullResilienceData` isn't immediately apparent as to what purpose it serves or when it would be used.

Reply:
Updating.